Add info about requestPrincipals creation (#324)
kyma-project/api-gateway: docs/istio-jwt.md

            - requiredScopes: ["read", "write"]
```

### Authentications

> Under the hood, authentications array will create corresponding [requestPrincipals](https://istio.io/latest/docs/reference/config/security/authorization-policy/#Source) array in Istio Authorization Policy resource.

Reviewer: ```suggestion
Under the hood, an authentications array creates a corresponding [requestPrincipals](https://istio.io/latest/docs/reference/config/security/authorization-policy/#Source) array in the Istio Authorization Policy resource.
```

> Every `requestPrincipals` string will be formatted as `<ISSUSER>/*`.

Reviewer: ```suggestion
Every `requestPrincipals` string is formatted as `<ISSUSER>/*`.
```

### Authorizations

> The authorizations field is optional. If it's not defined, the authorization will be satisfied if the JWT is valid. 

Reviewer: ```suggestion
The authorizations field is optional. When not defined, the authorization is satisfied if the JWT is valid. 
```

> Multiple authorizations can be defined for an access strategy. If there are multiple authorizations defined, the request will be allowed if at least one of the authorizations is satisfied.

Reviewer: ```suggestion
You can define multiple authorizations for an access strategy. When multiple authorizations are defined, the request is allowed if at least one of them is satisfied.
```
